src/actions/vstudio/vs2005.lua

```diff
@@ -51,16 +51,14 @@
 			if #user > 0 then
 				p.generate(prj, ".csproj.user", function() p.outln(user) end)
 			end
-
-		elseif premake.project.iscpp(prj) then
+		else
```

Comment:
Technically, this should probably have the check for C/C++, but I'm happy to ignore this since it's for VS2005 and VS2008, and VS2008 will be EOL in a little under a year.